CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+ 
+## Getting Started
+
+- To get an overview of the project, read the [README](README.md).
+- Make sure you have all the [prerequisites](README.md#prerequisites) so you can contribute to this project.
+
+### New Contributor Guide
+
+Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+## Conventions
+
+### Consider Starting The **Issue / Commit / PR Title** With An Applicable **Emoji**
+
+These are the conventional emojis for our project:
+
+Name|Emoji|Text Of Emoji To Provide|When To Use|
+|:---:|:---:|:---|:---|
+**FEATURE**|:rocket:|`:rocket:`|When working on a new feature.
+**DOC**|:memo:|`:memo:`|When writing docs.
+**BUG**|:bug:|`:bug:`|When fixing a bug.
+**STRUCT**|:file_folder:|`:file_folder:`|When rearranging files for better structure or orientation.
+**IMPROVE**|:horse_racing:|`:horse_racing:`|When improving performance of an existing feature.
+
+The *"Text Of Emoji To Provide"* should be the first word in the title.
+
+#### Examples
+
+- The PR title of:
+    ```
+    :memo: Updating `README.md`'s Description
+    ```
+    Will look like this:
+    ![](https://i.imgur.com/pNGetUm.png)
+
+- The commit title of:
+    ```
+    :memo: Adding A `CONTRIBUTING.md` File
+    ```
+    Will look like this:
+    ![](https://i.imgur.com/DfOnEPK.png)
+
+## Issues & PRs
+
+When contributing to this repository, please first discuss the change you wish to make via an issue.
+
+### Issues
+
+#### Create A New Issue
+If you spot a problem with the docs, search if an issue already exists. If a related issue doesn't exist, you can open a new issue.
+
+- #### Creating A New **Issue** That Contains **Many Tasks** Required To Be Done
+  If that is the case, then consider making a separate *"checkbox"* for each task.
+
+    #### Example
+    The issue content of:
+    ```
+    We need to complete the following tasks to be able to make coffee:
+      - [X] Buy coffee.
+      - [ ] Buy milk.
+      - [ ] Boil water.
+    ```
+    Will look like this:
+
+    ![](https://i.imgur.com/zr3acoh.png)
+
+#### Solve An Issue
+Scan through our existing issues to find one that interests you. You can narrow down the search using labels as filters, or a specific word in the issue title. If you find an issue to work on, you are welcome to open a PR with a fix.
+
+
+#### Commit Your Update
+
+Commit the changes once you are happy with them.
+
+Once your changes are ready, don't forget to self-review to speed up the review process :zap:.
+
+### Pull Requests (PRs)
```

Comment:
Nothing about Test Actions? should pass successfully? 
how to check locally that your code is ready to be pushed?

Minimum of TeamMembers Approvals?
For ideas, 
you could look here
https://docs.openstack.org/neutron/latest/contributor/policies/code-reviews.html#code-review

Reply:
I have updated the file in [6213999](https://github.com/redhat-beyond/JobSeeker/commit/621399924ef0c55d7971ff73ef460f7273254403). You may review.